app/js/components/LabOrdersList.jsx

```diff
@@ -1,559 +0,0 @@
-/* * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
- * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
- * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
- * graphic logo is a trademark of OpenMRS Inc.
- */
-import React, { PureComponent } from 'react';
-import R from 'ramda';
-import { connect } from 'react-redux';
-import cn from 'classnames';
-import moment from 'moment';
-import { withRouter } from 'react-router-dom';
-import PropTypes from 'prop-types';
-import { injectIntl, FormattedMessage } from 'react-intl';
-import swal from 'sweetalert';
-import { SortableTable, Loader, selectors } from '@openmrs/react-components';
-import LabOrderListFilters from './LabOrdersListFilters';
-import { fetchLabOrders, cancelOrder, printLabel } from '../actions/labOrdersAction';
-import { setSelectedConcept } from '../actions/labConceptsAction';
-import { filterThrough, calculateTableRows, getConceptShortName, computeResultStatus, isCancelable } from '../utils/helpers';
-import { loadGlobalProperties, selectProperty } from '../utils/globalProperty';
-import filtersAction from '../actions/filtersAction';
-import patientAction from '../actions/patientAction';
-import { getLabOrderables } from '../actions/labOrderablesAction';
-import { DEFAULT_ORDERS_BATCH_SIZE, DEFAULT_TABLE_PAGE_SIZE, FULFILLER_STATUS } from '../constants';
-import "../../css/lab-orders-list.scss";
-
-
-const Cell = ({ columnName, value, handleCancel, cancelMsg, enableLabelPrinting, handlePrint, printMsg, locale }) => {
```

Comment:
Nope, as I wrote in the description, the first commit is just moving files around. `App.js` was actually totally deleted because it was dead code. The second commit contains the functional changes.

Reply:
Ah, okay, App,js was just deleted, but the moving around was basically moving LabOrdersList.jsx into 3 new files,  LabOrdersList/index/jsx, Cell.jsx and DraftOrderTable.jsx, correct? Were there any significant changes to take a look at as part of that move?  It's hard to tell from a diff since it's moved into completely new files.

Reply:
> but the moving around was basically moving LabOrdersList.jsx into 3 new files, LabOrdersList/index/jsx, Cell.jsx and DraftOrderTable.jsx

Other stuff got moved too, but mostly, yes

> Were there any significant changes to take a look at as part of that move

Nope, as I wrote once in the description and once in the comment above, the first commit is just moving files around.